src/core/utils/useNotificationMessage.ts

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+
+type UseNotificationOptionsType = {
+  timeout?: number;
+  scrollToTop?: boolean;
+};
+type UseNotificationReturnType = [string | null, (text: string) => void];
+
+export const useNotificationMessage = ({
+  timeout,
```

Comment:
Great with the timeout. I like it!
But I think we should have a default so we do not need to think about what the timeout should be everytime we use the hook

Reply:
Same goes for scrollToTop. I think most of the time we want it to scroll to top? Or what do you think?